internal/nodes/avi_model_advl4_translator.go

```diff
@@ -165,8 +167,28 @@ func (o *AviObjectGraph) ConstructAdvL4VsNode(gatewayName, namespace, key string
 	// configures VS and VsVip nodes using infraSetting object (via CRD).
 	buildWithInfraSetting(key, namespace, avi_vs_meta, vsVipNode, infraSetting)
 
+	if len(serviceNSNames) > 0 {
+		svcNSName := strings.Split(serviceNSNames[0], "/")
+		service, err := utils.GetInformers().ServiceInformer.Lister().Services(svcNSName[0]).Get(svcNSName[1])
+		if err != nil {
+			utils.AviLog.Warnf("key: %s, msg: Error while retrieving service for gateway %s, skipping L4 Rule reconciliation", key, err.Error())
+		} else {
+			if traffic_disabled, ok := service.GetAnnotations()[lib.VSTrafficDisabled]; ok && strings.ToLower(traffic_disabled) == "true" {
+				utils.AviLog.Infof("key: %s, msg: Disable Traffic annotation is set, traffic will be disabled for service %s", key, service.Name)
+				avi_vs_meta.TrafficEnabled = proto.Bool(false)
```

Comment:
should we set default value to True?

Reply:
I think TrafficEnabled is by default set to True unless explicitly specified otherwise.